packages/web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/web-components",
-  "version": "4.53.0",
+  "version": "4.54.0",
```

Comment:
The original PR had this as a `patch`.

Reply:
yeah, I wasn't sure which, I can change it to a patch